README.md

```diff
@@ -1,2 +1,232 @@
+
+Date | Event | Slides | Video
+--- | --- | --- | ---
+Dec 2019 | WP Leiden meetup | [Slides](https://speakerdeck.com/jrf/your-code-can-be-poetry-too) |
+Mar 2018 | WordCamp Rotterdam | [Slides](https://speakerdeck.com/jrf/leveraging-the-wordpress-coding-standards-to-review-plugins-and-themes-1) | [Video](https://wordpress.tv/2018/04/18/juliette-reinders-folmer-leveraging-the-wordpress-coding-standards-to-review-plugins-and-themes/)
+Nov 2017 | WordCamp Utrecht | [Slides](https://speakerdeck.com/jrf/leveraging-the-wordpress-coding-standards-to-review-plugins-and-themes) | [Video](https://wordpress.tv/2018/01/05/juliette-reinders-folmer-leveraging-the-wordpress-coding-standards-to-review/)
+Sep 2017 | WP Fryslân meetup | [Slides](https://speakerdeck.com/jrf/for-non-developers) |
+
+
+## Installation
+
+### Requirements
+
+* PHP 5.4+.
+* [Composer](https://getcomposer.org/download/)
+
+### Installation for a specific project
+
+> #### About project based installation:
+> Using this method, the tooling will only be available to that specific project.
+>
+> **Pros**: Using a project based install, you document what tooling the project uses and make it easy for other contributors to the project to install that tooling.
+>
+> **Cons:** For each additional project, you will need to do the installation again.
+
+From the project root, run the following command:
+```bash
+composer require --dev jrfnl/qawpprojects
+```
+
+The PHP_CodeSniffer command will now be available from the project root as `vendor/bin/phpcs`.
+
+To update the install in the future, run the following command from the project root:
+```bash
+composer update jrfnl/qawpprojects --with-dependencies
+```
+
+### Global installation
+
+> #### About global installation:
+> Using this method, the tooling will be available from anywhere on your system.
+>
+> **Pros**: You only need to install it once.
+>
+> **Cons:** If you use the tooling for your projects and the projects are open to other contributors, it will be unclear what tooling you expect them to use.
+
+Run the following command from anywhere on your system:
+```bash
+composer global require --dev jrfnl/qawpprojects
+```
+
+Make sure the Composer "home" `vendor/bin` directory is in your system path.
+To find out what the Composer "home" directory is, run `composer config --list --global` and see what's listed under `home`.
+Take that directory, add `/vendor/bin` to it and make sure it's in your operating system's `$PATH` variable.
+
+The PHP_CodeSniffer command will now be available from anywhere on your computer as `phpcs`.
+
+To update the install in the future, run the following command from anywhere on your system:
+```bash
+composer global update jrfnl/qawpprojects --with-dependencies
+```
+
+### Verify the install succeeded
+
+Once the installation is finished, run the following command to verify the installation was succesfull:
+```bash
+# For a project based install:
+vendor/bin/phpcs -i
+
+# For a global install:
+phpcs -i
```

Comment:
This may not work out of the box, the composer `vendor/bin` isn't necessarily in the default PATH.

Reply:
I address that in the [Global Installation](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#global-installation) instructions.

> Make sure the Composer "home" vendor/bin directory is in your system path. To find out what the Composer "home" directory is, run composer config --list --global and see what's listed under home. Take that directory, add /vendor/bin to it and make sure it's in your operating system's $PATH variable.

If the command doesn't work, it basically means people didn't follow the installation instructions.